crates/utils/src/settings/mod.rs

```diff
@@ -60,7 +63,15 @@ impl Settings {
     let conf = self.database();
     format!(
       "postgres://{}:{}@{}:{}/{}",
-      conf.user, conf.password, conf.host, conf.port, conf.database,
+      conf
+        .user
+        .unwrap_or_else(|| DEFAULT_DATABASE_USER.to_string()),
+      conf.password,
+      conf.host,
+      conf.port.unwrap_or(DEFAULT_DATABASE_PORT),
+      conf
+        .database
+        .unwrap_or_else(|| DEFAULT_DATABASE_DB.to_string()),
```

Comment:
You should just be able to use `unwrap_or_default()`, and then there's no need to make these globals.

Reply:
Default for string is an empty string, and default for i32 is 0, so thats not what we want. I did try with `let defaults = DatabaseConfig::default()`, but that needs a lot of unwrap everywhere because the values are options. And it results in a lint error which I couldnt manage to fix (only ignore).

Reply:
I'll play with a few things in a bit.